extensions/wikia/AdEngine/js/spec/EvolveHelper.spec.js

```diff
@@ -48,30 +48,54 @@ describe('EvolveHelper', function () {
 		adContextMock.targeting.wikiDbName = null;
 		adContextMock.targeting.wikiCustomKeyValues = null;
 		adContextMock.targeting.wikiVertical = null;
-		evolveHelper = modules['ext.wikia.adEngine.evolveHelper'](adContextModuleMock, mockAdLogicPageParams(), mockDartUrl(), mockKrux(), logMock);
+		evolveHelper = modules['ext.wikia.adEngine.evolveHelper'](
+			adContextModuleMock,
+			mockAdLogicPageParams(),
+			mockDartUrl(),
+			mockKrux(),
+			logMock
+		);
 
 		expect(evolveHelper.getSect()).toBe('ros', 'ros');
 
 		adContextMock.targeting.wikiCustomKeyValues = 'foo=bar;media=tv';
 		adContextMock.targeting.wikiVertical = 'Entertainment';
-		evolveHelper = modules['ext.wikia.adEngine.evolveHelper'](adContextModuleMock, mockAdLogicPageParams(), mockDartUrl(), mockKrux(), logMock);
+		evolveHelper = modules['ext.wikia.adEngine.evolveHelper'](
+			adContextModuleMock,
+			mockAdLogicPageParams(),
+			mockDartUrl(),
+			mockKrux(),
+			logMock
+		);
 
 		expect(evolveHelper.getSect()).toBe('tv', 'tv entertainment');
 
 		adContextMock.targeting.wikiCustomKeyValues = 'foo=bar';
 		adContextMock.targeting.wikiVertical = 'Entertainment';
-		evolveHelper = modules['ext.wikia.adEngine.evolveHelper'](adContextModuleMock, mockAdLogicPageParams(), mockDartUrl(), mockKrux(), logMock);
+		evolveHelper = modules['ext.wikia.adEngine.evolveHelper'](
+			adContextModuleMock,
+			mockAdLogicPageParams(),
+			mockDartUrl(),
+			mockKrux(),
+			logMock
+		);
 
 		expect(evolveHelper.getSect()).toBe('entertainment', 'foo entertainment');
 
 		adContextMock.targeting.wikiCustomKeyValues = 'foo=bar;media=movie';
 		adContextMock.targeting.wikiVertical = 'Entertainment';
-		evolveHelper = modules['ext.wikia.adEngine.evolveHelper'](adContextModuleMock, mockAdLogicPageParams(), mockDartUrl(), mockKrux(), logMock);
+		evolveHelper = modules['ext.wikia.adEngine.evolveHelper'](
```

Comment:
We could extract it to `getModule()` as we got used to in our tests ;-) Small task ticket? :P